cypress/e2e/models/migration/reports/reports.ts

```diff
@@ -84,3 +61,11 @@ export function verifyApplicationRisk(risktype: string, appName: string): void {
             }
         });
 }
+export function closeArticle(articleTitle: string): void {
+    cy.get(articleTitle)
+        .closest(articleCard)
+        .find(articleHeader)
+        .within(() => {
+            cy.get("button").click();
```

Comment:
You can use the `button` constant we have as you're already doing in line 44

Reply:
changed line 44 to "articleButton" instead of button and line 69 to button from constants to avoid confusion (they are not the same constant)